lib/github/middleware.js

```diff
@@ -80,12 +80,15 @@ module.exports = function middleware (callback) {
     context.sentry.setTag('transaction', `webhook:${context.name}.${context.payload.action}`)
     for (let subscription of subscriptions) {
       const jiraHost = subscription.jiraHost
-
       context.sentry.setTag('jiraHost', jiraHost)
       context.sentry.setTag('gitHubInstallationId', gitHubInstallationId)
       context.sentry.setUser({ jiraHost, gitHubInstallationId })
-
       context.log = context.log.child({ jiraHost })
+      if (context.timedout) {
```

Comment:
Timing out needs to be handled cooperatively. So we have to opt-in and check if we're timed out each time.